games-arcade/sdlinvaders/sdlinvaders-0.7.6.recipe

```diff
@@ -42,13 +42,7 @@ BUILD_PREREQUIRES="
 	"
 	
 BUILD() {
-	if [ $targetArchitecture == "x86"  ] ||  [ $targetArchitecture == "x86_gcc2" ]; then
-		# for some reason, the x86 build doesn't like clang
-		make
-	else
-		# use clang instead of gcc, as gcc fails to compile
-		make CC=clang
```

Comment:
If we're now build with GCC you may as well remove cmd:clang from BUILD_PREREQUIRES.

Reply:
oh, I somehow forgot to remove that, sorry